deploy/buildtools/buildafi.py

```diff
@@ -21,6 +24,32 @@ def get_deploy_dir():
         deploydir = local("pwd", capture=True)
     return deploydir
 
+def sim_local(cmdline, capture=False):
+    """ setup environment and locally execute a cmdline in sim subdir"""
+    ddir = get_deploy_dir()
+    with prefix('cd ' + ddir + '/../'), \
+         prefix('export RISCV={}'.format(os.getenv('RISCV', ""))), \
+         prefix('export PATH={}'.format(os.getenv('PATH', ""))), \
+         prefix('export LD_LIBRARY_PATH={}'.format(os.getenv('LD_LIBRARY_PATH', ""))), \
+         prefix('source ./sourceme-f1-manager.sh'), \
+         prefix('cd sim/'), \
+         InfoStreamLogger('stdout'), \
+         InfoStreamLogger('stderr'):
+        return local(cmdline, capture=capture)
+
+def get_sim_makevar(buildconfig, varname):
+    """ Dump make database for buildconfig returning value for varname assignment """
+    cmdline = "{} | grep -E '^{} :?= '".format(
+        buildconfig.make_recipe('--dry-run --print-data-base'),
+        varname
+    )
+
+    res = sim_local(cmdline, capture=True)
+    var_results = res.stdout.splitlines()
+    assert len(var_results) == 1, "get_sim_makevar() command " + \
+           res.real_command + " STDOUT is not a single line. It was:\n" + pformat(var_results)
+    return var_results[0].split(None, 2)[-1]
+
 def replace_rtl(conf, buildconfig):
```

Comment:
rename to replace_rtl_local and update teh comment?

Reply:
I understand the desire to rename but @davidbiancolin what were you thinking you wanted to update in teh comment?

Reply:
Something more to the effect of: runs the build locally instead of shipping a fat jar. The name will capture that though.

Reply:
Ok.  GH is also doing a great job of hiding the third line of the comment "THIS ALWAYS RUNS LOCALLY". ;)